brake/backends/cachebe.py

```diff
@@ -71,7 +78,11 @@ def limit(self, func_name, request,
             if ':ip:' in counter:
                 ratelimited_by = 'ip'
 
-            if counters[counter] > count:
+            current_count = counters[counter]
+            if isinstance(current_count, tuple):
```

Comment:
This is nice, I think it gracefully handles the case where you're upgrading from the existing cache system. However, I realize there's still a problem for existing (old) django installations that are expecting a scalar value. (i.e. since you're not going to redeploy all of your web front ends at precisely the same time, there's going to be a window for the ratelimit code to throw exceptions on the older versions.

The way I see it, we have two options: call this a major version upgrade and let clients deal with the fallout, or we can release an intermediate version that doesn't write your new value, but *does* check for a tuple value and continues as normal. Then people can upgrade to the intermediate version with no race conditions in the value, and upgrade to this, enhanced version with no troubles.

I kind of like the second option best.

@skorokithakis what do you think?